### DISCUSSION TITLE
Server app for Nvidia Shield

### DISCUSSION THREAD

**Discussion by Dhop84** [pymedusa/Medusa#7455] [Ideas & Feature requests]
I'd like to be able to run Medusa on the Nvidia shield. As I'm sure your aware shield is becoming more and more popular, it runs Plex and Emby servers brilliantly and you can even run the android version of duckietv (another automated TV show downloader) but it's not perfect. Medusa would be the main TV show downloader on Android and the first to have an app made specifically for the shield, opening up way more users of Medusa and puts it at the top of the automated TV show downloader world 👍

**Comment by p0psicles** [comment]
Okay then we need someone who has experience with porting python to Android?

**Comment by p0psicles** [comment]
I also think this has been requested already
